docs/source/new-docs/tutorials/2024-tutorial.rst

```diff
@@ -0,0 +1,464 @@
+
+Recommended refresher reading on :ref:`plugin-vocabulary` and
+:ref:`required-attributes` before moving ahead.
+
+.. _plugin-development-setup1:
+
+Product Plugin Development Initial Setup
+****************************************
+
+Before creating a new product for CLAVR-x Cloud-Top-Height, let's get the
+initial setup done:
+
+#. To develop a new GeoIPS plugin, first install :ref:`GeoIPS<linux-installation>`
+   and ensure that the ``geoips`` Python environment is enabled throughout this
+   tutorial.
+
+   .. code-block:: shell
+
+    mamba activate geoips  # activating Python environment
+
+   You will know the geoips conda environment is enabled if (geoips) appears before
+   your username in the command prompt.
+
+#. Install GeoIPS CLAVR-x package and verify the installation, as this is
+   essential for developing products with GeoIPS CLAVR-x.
+
+   .. code-block:: shell
+
+    git clone https://github.com/NRLMMD-GEOIPS/geoips_clavrx $GEOIPS_PACKAGES_DIR/geoips_clavrx  # clone the remote repository
+    pip install -e $GEOIPS_PACKAGES_DIR/geoips_clavrx  # install geoips_clavrx
+
+    $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_clavrx  # download test data for geoips-clavrx
+    $GEOIPS_PACKAGES_DIR/geoips_clavrx/tests/test_all.sh  # run tests to verify geoips-clavrx installation
+
+#. Set the following additonal environment variables which are specific to
+   your product plugin development.
+
+   .. code-block:: shell
+
+      export MY_PKG_NAME=<your package name>  # refer to the note below regarding your package name
+      export MY_PKG_DIR=$GEOIPS_PACKAGES_DIR/$MY_PKG_NAME  # your package directory
+      export MY_PKG_URL=<your package’s URL on version control platform(GitLab)>  # your package VCS url
+
+   .. NOTE::
+      Choose a package name that is all lowercase, begins with a letter, and
+      contains only letters, numbers, and underscores.
+
+#. Navigate to your product plugin directory and clone the example repository
+   of customized plugin development, `Template Basic Plugin
+   <https://github.com/NRLMMD-GEOIPS/template_basic_plugin/tree/main>`_. This
+   repository serves as a base for building a new plugin package that contains
+   one or more custom plugins.
+
+   .. code-block:: shell
+
+      cd $GEOIPS_PACKAGES_DIR  # go to your package directory
+      git clone --no-tags --single-branch $GEOIPS_REPO_URL/template_basic_plugin.git
+
+   .. NOTE::
+    If you're not able to move into the directory listed in the above code-block, verify
+    that the environment variables are set using the commands shown below. Otherwise,
+    check step three again and seek help if needed.
+
+    .. code-block:: shell
+
+      echo $MY_PKG_NAME :  # should reflect your package name
+      echo $MY_PKG_DIR  :  # should reflect merged path of $GEOIPS_PACKAGES_DIR/$MY_PKG_NAME
+
+#.  Owning tutorial template package: change it's name, set the git branch to
```

Comment:
“change it’s name” → “change its name” (remove the apostrophe for possessive “its”).